### PR TITLE
Added a check for pulseaudio install

### PR DESCRIPTION
**Pull request by aaaeka**

### CODE REVIEW

File: pactl/pactl.go [hunk 1]
```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/l3lackShark/audio-share-discord-linux/shell"
 )
 
+//Checks for pulseaudio
+func CheckForPulseaudio() bool {
+	raw := string(shell.Cmd("pactl --version", true))
+	if (raw != "") {
```
Comment (l3lackShark) [inline pactl/pactl.go]: we don't do parenthesis on if statements in Go

File: pactl/pactl.go [hunk 2]
```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/l3lackShark/audio-share-discord-linux/shell"
 )
 
+//Checks for pulseaudio
+func CheckForPulseaudio() bool {
+	raw := string(shell.Cmd("pactl --version", true))
```
Comment (l3lackShark) [inline pactl/pactl.go]: are you sure that this is correct way of checking this? Wouldn't `pulseaudio --version` or something like `ps aux | grep pulseaudio` be better to make sure that PA is currently running?

Comment (l3lackShark) [reply pactl/pactl.go]: Actually `pgrep pulseaudio` would be perfect here I think.

Comment (aaaeka) [reply pactl/pactl.go]: Yeah that's probably the better way to go about it

**Pull request merged**